v3/plugins/che-incubator/cpptools/0.1/meta.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 publisher: che-incubator
 name: cpptools
-version: 0.1
+version: "0.1"
```

Comment:
And this one has to be made a string because it's short, i.e. not something like 1.0.9, right?

Reply:
Yup, `0.1` could be parsed as a number, so it is. `1.0.9` cannot, so it isn't and defaults to string. Other fun cases
```yaml
version: 1.0    # => version: 1
version: 0xa    # => version: 10
version: 0b1001 # => version: 9
version: 0777   # => version: 511
```

Considering forcing all strings to be quoted, honestly.

Reply:
+1 to that